### ISSUE TITLE
Ice Crystal crashes with MorePlayerModels

### ISSUE THREAD

**Issue by Konkerior** [BobMowzie/MowziesMobs#647]
Well, ive been trying to figure out whats the issue with the Ice Crystal, turns out i found that it crashes when used in a pack that has the ''more player models'' mod. when i use it against another player, they crash (but i dont), but if im in F5 mode, i also crash (so, we all crash) i guess it has something to do with the animation or idk.

**Comment by BobMowzie** [comment]
Please send the crash log.

**Comment by Konkerior** [comment]
[crash-2022-08-14_11.58.12-client.txt](https://github.com/BobMowzie/MowziesMobs/files/9333855/crash-2022-08-14_11.58.12-client.txt)
i also took a quick video, it happens 100% of the times

**Comment by Konkerior** [comment]
https://user-images.githubusercontent.com/111206754/184549405-4f1cfe0f-3e9d-4c8b-bb98-729bf250a070.mp4

if i look at somebody their game crashes, i also crash if im in F5 mode.

**Comment by BobMowzie** [comment]
Okay. Did you also report this to MorePlayerModels?

**Comment by Konkerior** [comment]
nope, i actually didnt, i guess i should, but since the problem occurs specifically with the ice crystal, i thought of posting something here first, thanks in advance
